tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts

```diff
@@ -100,14 +100,14 @@ function areSeriesDataListEqual(
   });
 }
 
-interface RunIdAndPoints {
-  runId: string;
+interface PartialSeries {
+  seriesId: string;
```

Comment:
In this PR, calls like `runSelectionMap.get(seriesData.seriesId)` and `getRunDisplayName(runIdAndPoint.seriesId)` assume that seriesId and runId are exactly the same.  Even though this assumption changes in the followup, could we add a comment in this PR here, noting that each `seriesId` is equal to 1 unique `runId`?

Alternatively, this PR could preserve the `runId` name, and the followup can introduce `seriesId`s.